contrib/epee/src/http_auth.cpp

```diff
@@ -448,19 +448,20 @@ namespace
             }
           };
 
+          namespace SP = std::placeholders;
```

Comment:
There is a `using namespace std::placeholders` above. Remove that, and consider moving this up with the other namespace alias.

I'm partially against cotire if it requires changes to code that otherwise compiles with the gcc and clang targets. Presumably its generation routine leaked a boost header which annoyingly dumps `_1` into the open namespace causing ambiguity. Arguably I should've never used `_1` etc, since I knew this was an issue. Its just irritating that cotire's generation routine is somehow different than both gcc and clang - it means there's another hidden target to test the build against.

Reply:
The using is needed for other stuff. I can just move the SP= to the top ?